src/nyc_trees/apps/survey/templates/survey/survey.html

```diff
@@ -52,6 +52,29 @@
 
         <button class="btn btn-primary" id="submit-survey">Submit</button>
     </div>
+    <div class="modal fade" id="quit-popup" tabindex="-1" role="dialog" aria-labelledby="quit-popup-title" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close btn btn-plain pull-right" data-dismiss="modal">
+                        <span aria-hidden="true">×</span>
+                        <span class="sr-only">Close</span>
+                    </button>
+                    <h4 class="modal-title" id="quit-popup-title">Stop Mapping This Block</h4>
+                </div>
+                    <div class="modal-body">
+                        <div>I can no longer map because</div>
+                        <form id="quit-form" method="POST">
+                            <textarea id="quit-reason" style="width: 100%;"
+                                      placeholder="Offset tree bed, bad sidewalks, etc."></textarea>
+                        </form>
+                    </div>
+                    <div class="modal-footer">
+                        <button id="quit" type="submit" class="btn btn-primary">Stop Mapping</button>
+                    </div>
+            </div>
+        </div>
+    </div>
```

Comment:
`extra_content` is a direct child of `<body>`, which makes certain that styling changes to `base.html` don't push this modal inside of an element with a low z-index, causing this modal to appear below other elements.